components/scream/src/physics/common/physics_functions.hpp

```diff
@@ -78,6 +78,13 @@ struct Functions
   KOKKOS_FUNCTION
   static Spack polysvp1(const Spack& t, const bool ice);
 
+  //  compute saturation vapor pressure
+  //  polysvp1 returned in units of pa.
```

Comment:
minor comment, but probably better to say "saturation vapor pressure returned in units of pa" rather than polysvp since we're removing the name polysvp from SCREAM in this PR.